neo-collector/create/createOrUpdatePrimaryTables.py

```diff
@@ -0,0 +1,141 @@
+import psycopg2
+import requests
+import json
+import time
+import socket
+import dns.resolver
+import socket
+import os
+import requests
+import time
+
+resolver = dns.resolver.Resolver(configure=False)
+resolver.nameservers = ["208.67.222.222", "208.67.220.220", '8.8.8.8', '2001:4860:4860::8888',
+                        '8.8.4.4', '2001:4860:4860::8844']
+
+host = str(os.environ['PGHOST'])
+databasename = str(os.environ['PGDATABASE'])
+user = str(os.environ['PGUSER'])
+password = str(os.environ['PGPASSWORD'])
+
+connection_str = "dbname='{}' user='{}' host='{}' password='{}'".format(
+    databasename, user, host, password)
+dsn = "postgresql://{}:{}@{}/{}".format(user, password, host, databasename)
+
+print(connection_str)
+
+def get_coz_mainnet_json():
```

Comment:
We need a bit more validation at this stage. Some simple checks to make sure the JSON file is in the format we expect